src/utils/calculateDimension.js

```diff
@@ -0,0 +1,32 @@
+export default function calculateDimension(el, axis, borderBox = false, withMargin = false) {
+  if (!el) {
+    return 0;
+  }
+
+  const axisStart = axis === 'width' ? 'Left' : 'Top';
+  const axisEnd = axis === 'width' ? 'Right' : 'Bottom';
+
+  // Only read styles if we need to
+  const style = (!borderBox || withMargin) ? window.getComputedStyle(el) : null;
+
+  // Offset includes border and padding
+  const { offsetWidth, offsetHeight } = el;
+  let size = axis === 'width' ? offsetWidth : offsetHeight;
+
+  // Get the inner size
+  if (!borderBox) {
+    size -= (
+      parseFloat(style[`padding${axisStart}`]) +
```

Comment:
does this need to pivot based on RTL?

Reply:
I do not believe so (or at the least, `react-dates` currently still works as expected with the `isRTL` prop). However, I am pegging to the old version of RWS-IA for this change and will come back to the question of RTL support with the new interface in a follow-up PR. :)